src/components/Navbar/SidebarNav.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from "react";
+import ReactDOM from "react-dom";
+import { NavLink } from "react-router-dom";
+
+import useWidth from "../../hooks/useWidth";
+
+import { RemoveIcon } from "../../assets";
+
+const SidebarNav = ({ setIsClicked, clickHandler }) => {
+  const { width } = useWidth();
+
+  useEffect(() => {
+    if (width > 700) {
+      setIsClicked(false);
+    }
+  }, [width, setIsClicked]);
+
+  const removeSideBar = (e) => {
+    if (e.target.classList.contains("navbar-backdrop")) {
+      setIsClicked(false);
+    }
+  };
+
+  return ReactDOM.createPortal(
+    <div className='navbar-backdrop' onClick={removeSideBar}>
+      <div className='navbar-menu-items card-container'>
+        <ul>
+          <li className='nav-items-head flex flex-col'>
+            <div className='head-cta flex ai-center jc-between'>
+              <h3 className='head-title'>Hi There!</h3>
+
+              <button
+                onClick={() => clickHandler()}
+                style={{ background: "#ffff", borderRadius: "50%" }}
+                className='btn btn-icon'>
+                <img src={RemoveIcon} alt='remove_icon' />
+              </button>
+            </div>
+
+            <div className='flex flex-gap'>
+              <NavLink to='/login' className='link-btn font-b'>
+                Log In
+              </NavLink>
+              or
+              <a href='./' className='link-btn font-b'>
```

Comment:
Please donot use Anchor tag for internal linking

Reply:
i will replace it the Link